modules/rostests/apitests/gdi32/GetObject.c

```diff
@@ -701,11 +701,23 @@ Test_Region(void)
     DeleteObject(hRgn);
 }
 
+void Test_CursorIcon()
+{
+    ICONINFO info;
+    BITMAP bmp;
+    HCURSOR hCursor = LoadImage(NULL, MAKEINTRESOURCE(OCR_NORMAL), IMAGE_CURSOR, 0, 0, LR_DEFAULTSIZE|LR_SHARED);
```

Comment:
You're currently using the Wide (Unicode) variant of `GetObject`, so please use the Unicode variant for this one as well.

Reply:
You mean GetObject(...) call?